scripts/build.ps1

```diff
@@ -93,14 +93,15 @@ $env:MSBUILD_VERSION = "15.0"
 Write-Verbose "Setup build configuration."
 $TPB_Solution = "TestPlatform.sln"
 $TPB_TestAssets_Solution = Join-Path $env:TP_ROOT_DIR "test\TestAssets\TestAssets.sln"
-$TPB_TargetFramework = "net451"
+$TPB_TargetFramework45 = "net45"
+$TPB_TargetFramework451 = "net451"
 $TPB_TargetFramework472 = "net472"
-$TPB_TargetFrameworkCore20 = "netcoreapp2.1"
 $TPB_TargetFrameworkCore10 = "netcoreapp1.0"
+$TPB_TargetFrameworkCore20 = "netcoreapp2.1"
 $TPB_TargetFrameworkUap = "uap10.0"
-$TPB_TargetFrameworkNS2_0 = "netstandard2.0"
-$TPB_TargetFrameworkNS1_3 = "netstandard1.3"
-$TPB_TargetFrameworkNS1_0 = "netstandard1.0"
+$TPB_TargetFrameworkNS10 = "netstandard1.0"
```

Comment:
Any reason to replace the less ambiguous NS2_0 naming with NS20 naming?

Reply:
To keep all target framework variable names in same format.